packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -487,9 +482,19 @@ const ScrollAreaImpl = React.forwardRef(function ScrollAreaImpl(props, forwarded
   };
 
   return (
-    <DispatchContext.Provider value={dispatch}>
-      <ScrollAreaRefsContext.Provider value={refs}>
-        <ScrollAreaContext.Provider value={context}>
+    <DispatchProvider dispatch={dispatch}>
+      <ScrollAreaRefsProvider {...refs}>
+        <ScrollAreaProvider
```

Comment:
I can see the new provider here with all the props, but I can't see the `context` being deleted in the old one. What's up here?

Reply:
Ah crap, I must have hastily pushed with `--no-verify` on Friday, slapped wrists 🙈